src/components/Pagination/Pagination.js

```diff
@@ -1,4 +1,4 @@
-import React, { useCallback } from 'react';
```

Comment:
Let's put these changes in a different PR. This is a bug-fix to make Pagination work in ngui, while putting react in devDependencies is not.